dev/tools/create_overlays.sh

```diff
@@ -65,6 +65,7 @@ do
         _CONTRIB_GITPUSHURL="git@github.com:$DEVELOPER_NAME/${_CONTRIB_GITSUFFIX}.git"
     else
         skipped_repos="$skipped_repos $_CONTRIB_NAME"
+        shift
```

Comment:
Seems like it might be more future-proof to instead move the `shift` before to occur right after the setting of `_CONTRIB_NAME`